lib/plugins/features/movies/controller.js

```diff
@@ -3,7 +3,9 @@
 const Movie = require('../../../models/movie');
 
 exports.create = async (payload) => {
-  const movie = await new Movie().save(payload);
+  const tempPayload = Object.assign({}, payload);
+  tempPayload.name = tempPayload.title;
+  Reflect.deleteProperty(tempPayload, 'title');
```

Comment:
Can you just use [`delete`](https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Operators/delete)?

Reply:
update: linter doesn't allow `delete` lol